Samples/2.0/ChatRoom/OrleansServer/Program.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Net;
+using System.Runtime.CompilerServices;
+using GrainImplementation;
+using Microsoft.Extensions.Logging;
+using Orleans;
+using Orleans.Configuration;
+using Orleans.Hosting;
+using Orleans.Runtime;
+using Orleans.Runtime.Configuration;
+using Orleans.Streams;
+using Utils;
+
+namespace OrleansServer
+{
+	class Program
+	{
+		static void Main(string[] args)
+		{
+		    var builder = new SiloHostBuilder()
+		        .Configure<ClusterOptions>(options =>
+		        {
+		            options.ClusterId = Constants.ClusterId;
+		            options.ServiceId = Constants.ServiceId;
+		        })
+		        .UseLocalhostClustering()
+		        .Configure<EndpointOptions>(options => options.AdvertisedIPAddress = IPAddress.Loopback)
+                .ConfigureApplicationParts(parts => parts.AddApplicationPart(typeof(Channel).Assembly).WithReferences())
+		        .ConfigureLogging(logging => logging.AddConsole())
+                //need to configure a grain storage called "PubSubStore" for using streaming with ExplicitSubscribe pubsub type
+                .AddMemoryGrainStorage("PubSubStore")
+		        .AddSimpleMessageStreamProvider(Constants.ChatRoomStreamProvider); 
```

Comment:
Maybe we could also show here (with a commented out line) that one can easily replace `AddSimpleMessageStreamProvider` with `AddAzureQueueStreams` to get the messages go over Azure Queues instead?
What do you think?

Reply:
ok

Reply:
actually at a second thought, it might be confusing to do that, since it wouldn't be like "uncomment this, and the sample will just work", the project need to reference AzureQueueStreaming package, and add connection string, etc. I think I can just parse in the streaming doc link here and it will be much clearer to read the doc than doing some simple stuff here.